### ISSUE TITLE
Azure build pipeline issues.

### ISSUE THREAD

**Issue by hyunoosung** [jaredpalmer/razzle#1214]
1. With Azure DevOps pipeline, the process.env.PORT is locked at port 3000 which generates 3e3 hex value in server.js.

2. Once deployed, homepage generates below error.
Error: EINVAL: invalid argument, stat '/home/site/wwwroot/D:\a\1\s\build\public/index.html'

I have applied work around in https://github.com/jaredpalmer/razzle/issues/906 but the best result so far is as below.
            .use(o.a.static("D:\\a\\1\\s\\build\\public"))

"D:\\a\\1\\s\\build\\public" directory is where Azure devOps CI uses for build pipeline.

#1 can be resolved by just manually setting const port = 8080 since Azure WebApp uses 8080 port.
#2 I can only resolve this by using VSCode Azure App Service extension and manually change  .use(o.a.static("D:\\a\\1\\s\\build\\public")) to .use(o.a.static("build/public"))

Can anyone help to fix this build issue or point out where to look at?

**Comment by cloudhospital** [comment]
Got workaround of 2.
Changed 
.use(express.static(process.env.RAZZLE_PUBLIC_DIR!))
to .use(express.static('build/public'))

and the built server.js now has 
use(o.a.static("build/public"))

But I'm sure there should be definitely better approach with modifying RAZZLE_PUBLIC_DIR.

**Comment by nimaa77** [comment]
hello, what is the problem?
is that something about environment variables?
maybe this helps you https://github.com/jaredpalmer/razzle#runtime-variables

I'm living in a country that can't use Azure or AWS because of USA sanctions so I have no experience with azure

if you explain the problem more maybe I can help you with it

**Comment by hyunoosung** [comment]
It was process.env.RAZZLE_PUBLIC_DIR.
Azure DevOps CI pipeline build and drop artifact in vm which has physical address like I shared ex) D:/a/1/s/something...
Then Razzle build take this physical address into server.js and generates 'wwwroot/D:\a\1\s\build\public' for public dir.
where Azure web app just need public folder name as relative path for example 'build/public'
So I had to hardcord Server.ts as below.

.use(express.static('build/public'))
